manifest/kcp.yaml

```diff
@@ -105,12 +105,18 @@ spec:
         - --tls-private-key-file=/etc/kcp/tls/server/tls.key
         - --tls-cert-file=/etc/kcp/tls/server/tls.crt
         - --root-directory=/tmp/kcp
+        - --run-virtual-workspaces=false
+        - --virtual-workspace-address=https://$(EXTERNAL_HOSTNAME)
```

Comment:
no port?

Reply:
or is the front proxy doing the magic?

Reply:
Since this is exposed using an Openshift route for now, the port (443) is implied.  If we want to make this template more generic for use outside of our shared service, we would need to do more work to generalize.